src/main/java/frc/robot/swerve/SwerveDrive.java

```diff
@@ -258,6 +260,23 @@ public Command driveCommand(
                                 fieldOriented.getAsBoolean()));
     }
 
+    public Command turnCommand(double rotation, double turnTolerance) {
+        PIDController turnController =
```

Comment:
Better to use a motion profile.

Reply:
Eitan said Dieter controller works best for rotations